### DISCUSSION TITLE
buddhism/2024/books/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#105] [General]
# buddhism/2024/books/

https://shanenull.com/buddhism/2024/books/

**Comment by shane0** [comment]
How did I forget this one they also have an app

https://84000.co/


Guru & mantra

https://youtu.be/7Covu0rQU8I

**Reply by shane0** [reply]
# lions roar

- auction https://auction.lionsroar.com/lite-ui/?controller=home

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/books/#read

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2022/books/

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/books/

**Comment by shane0** [comment]
Srimala sutra
https://youtu.be/SOYlzJjSQxc?si=eO1cYDI6l8VNqXhn

**Comment by shane0** [comment]
# sutra study

sutra on impermanence 
https://youtu.be/5u4SEkcL9ZQ?si=Q6Twp1looVQEvezG

**Comment by shane0** [comment]
# lotus sutra

**Reply by shane0** [reply]
https://youtu.be/NWsfCOciyAU?si=7tVa234Zn2_0AHiM

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2023/lotus_sutra/

**Reply by shane0** [reply]
“THE HEART
* OF THE A
LOTUS SUTRA
LECTURES ON THE "EXPEDIENT MEANS"
AND "LIFE SPAN" CHAPTERS
DAISAKU IKEDA”

**Reply by shane0** [reply]
https://www.dharmawheel.net/viewtopic.php?t=35754

**Comment by shane0** [comment]
# chan

https://www.bdkamerica.org/product/three-chan-classics/

**Comment by shane0** [comment]
http://shanenull.com/buddhism/2024/quotes/

**Comment by shane0** [comment]
https://2024-buddhism.gitbook.io/buddhism-gitbook-2024

**Comment by shane0** [comment]
# 10k lines 25k lines

- 10k https://84000.co/translation/toh11
- 25k ?

**Reply by shane0** [reply]
The "Prajnaparamita in 25,000 Lines" was translated into English by several scholars, including **G. T. Stcherbatsky** and **N. H. L. P. V. P. M. Tsongkhapa**. More recent translations and studies have also been conducted by scholars like **Donald Lopez** and others who focus on Mahayana texts. Each translation may vary in style and interpretation, reflecting the complexities of the text. If you need more details about a specific translator or their work, feel free to ask!

**Reply by shane0** [reply]
https://www.dharmawheel.net/viewtopic.php?t=35754

**Reply by shane0** [reply]
- 25k lines https://84000.co/translation/toh9
